packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -193,16 +206,40 @@ describe('<Tooltip />', () => {
 
   describe('prop: delay', () => {
     it('should take the enterDelay into account', () => {
-      const wrapper = mount(<Tooltip enterDelay={111} {...defaultProps} />);
+      const wrapper = mount(<Tooltip {...defaultProps} enterDelay={111} />);
       simulatePointerDevice();
       const children = wrapper.find('#testChild');
-      focusVisible(children);
+      focusVisibleLegacy(children);
       assert.strictEqual(wrapper.find('[role="tooltip"]').exists(), false);
       clock.tick(111);
       wrapper.update();
       assert.strictEqual(wrapper.find('[role="tooltip"]').exists(), true);
     });
 
+    it('should use hysteresis with the enterDelay', () => {
```

Comment:
@eps1lon Oh boy, it's in the contrast of this file that I truly realize how tests are written with `@test-library/react` are much better than the one we have with enzyme 👏.